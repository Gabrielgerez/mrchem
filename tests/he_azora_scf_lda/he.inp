world_prec = 1.0e-4

WaveFunction {
  method = LDA
  restricted = true
  relativity = azora
}


Molecule {
$coords
He   0.0 0.0 0.0
$end
}
