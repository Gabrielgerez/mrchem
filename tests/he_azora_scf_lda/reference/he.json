{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "he",
          "r_rms": 3.5849373401e-05,
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": false,
      "omp_threads": -1,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 6,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "he",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.0001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.0001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.0001,
          "shared_memory": false,
          "smooth_prec": 0.0001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        },
        "zora_operator": {
          "include_coulomb": false,
          "include_nuclear": false,
          "include_xc": false,
          "isAZORA": true
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "relativity": "AZORA",
        "restricted": true,
        "rotate": true,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.0001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "environment": "None",
        "external_field": "None",
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.0001,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (SVWN5)",
        "orbital_thrs": 0.001,
        "relativity": "AZORA",
        "rotation": 0,
        "start_prec": 0.0001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.0
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.5414130052328172e-12,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.0
          ],
          "vector_el": [
            0.0,
            0.0,
            0.0
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "He",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.570464666039006
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -1.140929332078012
      },
      "scf_energy": {
        "E_ee": 1.9962222364216329,
        "E_eext": 0.0,
        "E_el": -2.835003921655858,
        "E_en": -6.625820552295827,
        "E_kin": 2.7679525130939138,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": 0.0,
        "E_tot": -2.835003921655858,
        "E_x": 0.0,
        "E_xc": -0.9733581188755775,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 2.0262673615492934,
        "E_eext": 0.0,
        "E_el": -2.8061198579393087,
        "E_en": -6.595209258134876,
        "E_kin": 2.7470772638332845,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": 0.0,
        "E_tot": -2.8061198579393087,
        "E_x": 0.0,
        "E_xc": -0.9842552251870108,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 1.973384349083191,
              "E_eext": 0.0,
              "E_el": -2.8339954491064434,
              "E_en": -6.528192694988571,
              "E_kin": 2.683318941139394,
              "E_next": 0.0,
              "E_nn": 0.0,
              "E_nuc": 0.0,
              "E_tot": -2.8339954491064434,
              "E_x": 0.0,
              "E_xc": -0.9625060443404567,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -2.8339954491064434,
            "energy_update": 0.027875591167134672,
            "mo_residual": 0.0708364444278362,
            "wall_time": 1.743911586
          },
          {
            "energy_terms": {
              "E_ee": 1.994312226962929,
              "E_eext": 0.0,
              "E_el": -2.8349265026797847,
              "E_en": -6.6081583679284135,
              "E_kin": 2.7511471759544044,
              "E_next": 0.0,
              "E_nn": 0.0,
              "E_nuc": 0.0,
              "E_tot": -2.8349265026797847,
              "E_x": 0.0,
              "E_xc": -0.9722275376687047,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -2.8349265026797847,
            "energy_update": 0.0009310535733413339,
            "mo_residual": 0.008304451068131975,
            "wall_time": 1.454814606
          },
          {
            "energy_terms": {
              "E_ee": 1.9954809275242602,
              "E_eext": 0.0,
              "E_el": -2.834996526943462,
              "E_en": -6.620132609175466,
              "E_kin": 2.7626021706049264,
              "E_next": 0.0,
              "E_nn": 0.0,
              "E_nuc": 0.0,
              "E_tot": -2.834996526943462,
              "E_x": 0.0,
              "E_xc": -0.9729470158971822,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -2.834996526943462,
            "energy_update": 7.002426367730408e-05,
            "mo_residual": 0.0016099564962614482,
            "wall_time": 1.523955258
          },
          {
            "energy_terms": {
              "E_ee": 1.9962222364216329,
              "E_eext": 0.0,
              "E_el": -2.835003921655858,
              "E_en": -6.625820552295827,
              "E_kin": 2.7679525130939138,
              "E_next": 0.0,
              "E_nn": 0.0,
              "E_nuc": 0.0,
              "E_tot": -2.835003921655858,
              "E_x": 0.0,
              "E_xc": -0.9733581188755775,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -2.835003921655858,
            "energy_update": 7.394712395747405e-06,
            "mo_residual": 0.0005919718940979519,
            "wall_time": 1.566191235
          }
        ],
        "wall_time": 6.289502326
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
