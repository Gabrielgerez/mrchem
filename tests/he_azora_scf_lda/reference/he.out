                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         azora_all_potentials                     ***
***         Git commit hash    fdae9976706e98637414                     ***
***         Git commit author  moritzgubler                             ***
***         Git commit date    Tue Oct 22 11:38:42 2024 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : HEAD
 Git commit hash       : 720133372c9717134c5a
 Git commit author     : gitpeterwind
 Git commit date       : Thu Feb 15 13:44:31 2024 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 6
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      He            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999999998814e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Relativity              : AZORA
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  2.747077263833
 E-N energy              :            (au)                 -6.595209258135
 Coulomb energy          :            (au)                  2.026267361549
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -0.984255225187
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -2.806119857939
 Nuclear energy          :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.806119857939e+00
                         :      (kcal/mol)             -1.760866796213e+03
                         :        (kJ/mol)             -7.367466675357e+03
                         :            (eV)             -7.635841130692e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.542683110301
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.085366220603
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (SVWN5)
 Relativity              : AZORA
 Environment             : None
 External fields         : None
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-04
 Final precision         : 1.00000e-04
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-03
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00          -2.806119857939         -2.806120e+00
    1          7.083644e-02          -2.833995449106         -2.787559e-02
    2          8.304451e-03          -2.834926502680         -9.310536e-04
    3          1.609956e-03          -2.834996526943         -7.002426e-05
    4          5.919719e-04          -2.835003921656         -7.394712e-06
---------------------------------------------------------------------------
                      SCF converged in 4 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      He            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  2.767952513094
 E-N energy              :            (au)                 -6.625820552296
 Coulomb energy          :            (au)                  1.996222236422
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -0.973358118876
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -2.835003921656
 Nuclear energy          :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.835003921656e+00
                         :      (kcal/mol)             -1.778991819845e+03
                         :        (kJ/mol)             -7.443301774231e+03
                         :            (eV)             -7.714438672107e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.570464666039
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.140929332078
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  7s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
